### ISSUE TITLE
CRITICAL BUG: Calling PDF or PNG Export Function makes redrawing of chart impossible

### ISSUE THREAD

**Issue by skuske** [highcharts/highcharts-ios#396]
@ihnatmoisieiev 

I found a serious issue today making the wrapper unusable once you call an export method: charts no longer draw then.

Steps to reproduce:

```
1. Create a chart
2. Call 'exportToPDF' via code
3. Redraw the chart ---- THIS FAILS
```

See attached sample project:

```
1. Start the project
2. Tap the 'Export PDF' button
3. Close the export sheet
4. Tap the 'Redraw' button
>>>> Chart fails to draw
```

Observe the debug console in Xcode during redraw of the chart. It will display a huge list of missing plugins.

<img width="598" alt="xcode-errors" src="https://user-images.githubusercontent.com/12996815/161418771-4835a504-4861-476d-a319-16fc9046ae0d.png">

Please fix that asap as this is a serious issue making the whole chart generation impossible once an export function was used.

Sample project attached. 

Thanks.

[sample.zip](https://github.com/highcharts/highcharts-ios/files/8404047/sample.zip)

**Comment by skuske** [comment]
@ihnatmoisieiev 

Additional info: the issue seems to occur only if you use `exportToPDF` or `exportToPNG`, but **not** `exportToCSV`.

And: to make chart generation possible again, the app must be closed and reopened.

**Comment by skuske** [comment]
@ihnatmoisieiev

Any comment on this please?

**Comment by ihnatmoisieiev** [comment]
@skuske thank you for the reporting. We are investigating the issue and will fix it as soon as possible.

**Comment by ihnatmoisieiev** [comment]
@skuske fixed, ` v10.0.0.1`. Please check and let us know.

**Comment by skuske** [comment]
@ihnatmoisieiev 

Fixed. Hopefully this was the last export related issue. ;o)
